### ISSUE TITLE
Tests: add API categories to test

### ISSUE THREAD

**Issue by vinyldarkscratch** [foolip/mdn-bcd-collector#572]
Something that came to mind is that a lot of times, when running individual tests, I end up testing similar APIs based upon their spec.  I'm thinking we could add additional test endpoints for collections of APIs based upon their spec.  It should be relatively easy using the Webref and the custom IDL filenames!

**Comment by foolip** [comment]
Yeah, adding a "spec" with the shortname to each test in test.json would allow this, and should be fairly straightforward. Most of the work will be in the flattening of IDL, to keep track of what came from where.
